app/views/spotlight/about_pages/_empty.html.erb

```diff
@@ -0,0 +1,22 @@
+<h2>Building this about page</h2>
```

Comment:
No, I don't think so. It would have the page title, as before, but not the heading or steps that are part of this ticket.